assets/js/components/Event.jsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Card from './spectre/Card'
 
-const Event = ({title}) => <div>{title}</div>
+const Event = ({title, subtitle, description, thumbnail}) => <Card>
+  <Card.Image>
+    <img className="img-responsive" src={thumbnail} />
```

Comment:
what's the point of having a `<Card.Image>` component if you can't just pass it a src and be done with it?

Reply:
:+1: